File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/LedgerConsensusException.java
package org.hyperledger.indy.sdk;

public class LedgerConsensusException extends IndyException
{
	private final static String message = "No consensus was reached during the ledger operation.";

	LedgerConsensusException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/LedgerApi/LedgerConsensusException.cs
namespace Hyperledger.Indy.LedgerApi
{
    /// <summary>
    /// Exception thrown when the no consensus was reached during a ledger operation.
    /// </summary>
    public class LedgerConsensusException : IndyException
    {
        const string message = "No consensus was reached during the ledger operation";

        /// <summary>
        /// Initializes a new LedgerConsensusException.
        /// </summary>
        internal LedgerConsensusException() : base(message, (int)ErrorCode.LedgerNoConsensusError)
        {

        }
    }

}
